tests/test_wave_01.py

```diff
@@ -43,9 +43,9 @@ def test_create_no_genre_movie():
     new_movie = create_movie(movie_title, genre, rating)
 
     # Assert
-    assert new_movie is None
+    assert new_movie == None
```

Comment:
Great; this check works, though it is more customary to use `is` to check whether a variable is None. [This article](https://www.pythontutorial.net/advanced-python/python-none/) gives a good explanation.